rex_xai/distributions.py

```diff
@@ -65,8 +65,8 @@ def str2distribution(d: str) -> Distribution:
 def random_coords(d: Optional[Distribution], *args, map=None) -> int:
     """generates random coordinates given a distribution and args"""
 
-    if args[0] < 2:
-        return -1
+    # if args[0] < 2:
+    #     return -1
```

Comment:
These lines can probably be deleted?

Also, this function returns -1 if the distribution isn't one of the implemented options - would this be a good use case for `ReXError`, or is the -1 numerically meaningful here?